app/controllers/caterings_controller.rb

```diff
@@ -9,6 +9,7 @@ def show
 		rescue ActiveRecord::RecordNotFound
 			record_not_found()
 		end
+    @daily_meals_today = DailyMeal.where("catering_id = ? AND serving_day = ?" ,@catering.id, Date.today).take(10)
```

Comment:
Now you see why spaces and tabs don't mix :japanese_goblin: :laughing:   
Is there any reason to write this query as ~raw SQL? `DailyMeal.where(catering: @catering, serving_day: Date.today)` would probably work as well.
Also, I'd remove that `.take(10)` call.